util/applications/src/iScience/generate_reads/cmd_arguments.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2006-2024 Knut Reinert & Freie Universität Berlin
+// SPDX-FileCopyrightText: 2016-2024 Knut Reinert & MPI für molekulare Genetik
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <filesystem>
+#include <vector>
+
+namespace raptor::util::generate_reads
+{
+
+enum class weight : uint8_t
+{
+    from_file_sizes,
+    from_hll_sketches,
+    from_kmer_counts,
+    from_weight_column,
+    from_uniform_distribution
```

Comment:
There is another cpp that generates reads (for RNA-seq) right? If we have the uniform distribution included here, do we need the other file? (doesn't have to be part of this PR or happen at all, just thinking out loud).

Reply:
> There is another cpp that generates reads (for RNA-seq) right?

Not that I know of